app/src/main/java/ch/epfl/qedit/view/edit/EditOverviewFragment.java

```diff
@@ -60,17 +64,39 @@ public String getText(Question item) {
                                 return item.getTitle();
                             }
                         });
+    }
 
+    private void setListener() {
         adapter.setItemListener(
                 new ListEditView.ItemListener() {
                     @Override
                     public void onItemEvent(int position, ListEditView.EventType type) {
-                        if (type == ListEditView.EventType.RemoveRequest)
-                            adapter.removeItem(position);
+                        switch (type) {
+                            case Select:
+                                setFragment(R.id.question_details_container);
+                                model.getFocusedQuestion().postValue(position);
+                                break;
+                            case RemoveRequest:
+                                adapter.removeItem(position);
+                                break;
```

Comment:
Here, should we test if the FocusedQuestion is the one we remove? And if so set it to the previous question or a default value, i don't know.

Reply:
Yes I think there is indeed a problem here.

Reply:
You were right. I resolved the problem.